fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -658,10 +654,7 @@ public boolean visit(java.lang.String elementName, int elementIndex, Identifier
                         p = new TokenParmVal();
                         p.setResourceType(cp.getResourceType());
                         p.setName(SearchUtil.makeCompositeSubCode(cp.getName(), SearchConstants.OF_TYPE_MODIFIER_COMPONENT_TYPE));
-                        if (typeCoding.getSystem() != null) {
```

Comment:
Do we need this for visit(..., Reference) as well, around line 707, where it handles the ':identifier' suffix?

Reply:
yes, good catch - added logic here and in JDBCQueryBuilder to handle code system case-sensitivity for :identifier modifier